frontend/components/Header.js

```diff
@@ -162,7 +162,9 @@ class Header extends Component {
                       </li>
                       <li className="divider" />
                       <li>
-                        <a href=".">Medical recovery centre</a>
+                        <Link href="/medical-recovery-centre">
+                          <a>Medical recovery centre</a>
```

Comment:
Just title case bothering me for subheaders "Medical Recovery Centre"

Reply:
Ah, I'm a sucker for copy and pasting code. How I made it through Uni 101. Fixed!